server/rest/src/main/java/org/infinispan/rest/resources/ServerResource.java

```diff
@@ -71,55 +87,59 @@ public ServerResource(InvocationHelper invocationHelper) {
+            .permission(AuthorizationPermission.ADMIN).name("CONNECTOR START").auditContext(AuditContext.SERVER)
+            .handleWith(this::connectorStartStop)
             .invocation().methods(POST).path("/v2/server/connectors/{connector}").withAction("stop")
-               .permission(AuthorizationPermission.ADMIN).name("CONNECTOR STOP").auditContext(AuditContext.SERVER)
-               .handleWith(this::connectorStartStop)
+            .permission(AuthorizationPermission.ADMIN).name("CONNECTOR STOP").auditContext(AuditContext.SERVER)
+            .handleWith(this::connectorStartStop)
             .invocation().methods(GET).path("/v2/server/connectors/{connector}/ip-filter")
-               .permission(AuthorizationPermission.ADMIN).name("CONNECTOR FILTER GET").auditContext(AuditContext.SERVER)
-               .handleWith(this::connectorIpFilterList)
+            .permission(AuthorizationPermission.ADMIN).name("CONNECTOR FILTER GET").auditContext(AuditContext.SERVER)
+            .handleWith(this::connectorIpFilterList)
             .invocation().methods(POST).path("/v2/server/connectors/{connector}/ip-filter")
-               .permission(AuthorizationPermission.ADMIN).name("CONNECTOR FILTER SET").auditContext(AuditContext.SERVER)
-               .handleWith(this::connectorIpFilterSet)
+            .permission(AuthorizationPermission.ADMIN).name("CONNECTOR FILTER SET").auditContext(AuditContext.SERVER)
+            .handleWith(this::connectorIpFilterSet)
             .invocation().methods(DELETE).path("/v2/server/connectors/{connector}/ip-filter")
-               .permission(AuthorizationPermission.ADMIN).name("CONNECTOR FILTER DELETE").auditContext(AuditContext.SERVER)
-               .handleWith(this::connectorIpFilterClear)
+            .permission(AuthorizationPermission.ADMIN).name("CONNECTOR FILTER DELETE").auditContext(AuditContext.SERVER)
+            .handleWith(this::connectorIpFilterClear)
             .invocation().methods(GET).path("/v2/server/datasources")
-               .permission(AuthorizationPermission.ADMIN).name("DATASOURCE LIST").auditContext(AuditContext.SERVER)
-               .handleWith(this::dataSourceList)
+            .permission(AuthorizationPermission.ADMIN).name("DATASOURCE LIST").auditContext(AuditContext.SERVER)
+            .handleWith(this::dataSourceList)
             .invocation().methods(POST).path("/v2/server/datasources/{datasource}").withAction("test")
-               .permission(AuthorizationPermission.ADMIN).name("DATASOURCE TEST").auditContext(AuditContext.SERVER)
-               .handleWith(this::dataSourceTest)
+            .permission(AuthorizationPermission.ADMIN).name("DATASOURCE TEST").auditContext(AuditContext.SERVER)
+            .handleWith(this::dataSourceTest)
+            .permission(AuthorizationPermission.ADMIN).name("CONNECTOR FILTER DELETE").auditContext(AuditContext.SERVER)
+            .handleWith(this::connectorIpFilterClear)
```

Comment:
Can you remove the formatting changes here, it makes it very difficult to grok what the actually changes are when viewing the diff.

Reply:
It appears "CONNECTOR FILTER DELETE" has been duplicated by mistake.

Reply:
Oops, I messed up the rebase

Reply:
I've cleaned it up